### ISSUE TITLE
https://lsatcracked.com link doesn't work (I have to type www)

### ISSUE THREAD

**Issue by mwalsh2020** [mwalsh2020/lsat-cracked#82]
This is weird becuase I thought it was working.

But I clicked on one of my the links I put in my YouTube description (https://lsatcracked.com, because I thought it looked sleeker than the www. version, and it didn't work)

**Comment by caioertai** [comment]
I'll check then. But do it with http://lsatcracked.com
There's no need for the S. The app redirects the user to the secure version.

**Comment by caioertai** [comment]
I don't think that impacts as much or at all, @mwalsh2020. Let me know if you think otherwise.
